docs/source/gbd2020_models/risk_attributable_causes/wasting/index.rst

```diff
@@ -644,35 +644,33 @@ in terms of the following variables:
      - All category "prevalences" are scaled down, such that the prevalence of cat 0 (the reincarnation pool) and the prevalences of the wasting categories sum to 1
    * - :math:`mam_tx_coverage`
      - Proportion of MAM (CAT 2) cases that have treatment coverage
-     - 0.488
+     - 0.15 (95% CI: 0.1, 0.2)
      - Potentially to be updated
    * - :math:`sam_tx_coverage`
      - Proportion of SAM (CAT 1) cases that have treatment coverage
-     - 0.488
+     - 0.488 (95% CI: 0.374, 0.604)
      - Potentially to be updated
    * - :math:`time_to_mam_ux_recovery`
      - Without treatment or death, average days spent in MAM before recovery
      - 63
-     - Potentially to be updated
+     - 
    * - :math:`time_to_mam_tx_recovery`
      - With treatment and without death, average days spent in MAM before recovery
-     - 41.3
-     - Potentially to be updated
+     - 41.3 (95% CI: 34.4, 49)
+     - 
    * - :math:`time_to_sam_ux_recovery`
      - Without treatment or death, average days spent in SAM before recovery
-     - 60.5
-     - Potentially to be updated
+     - See the definition on the :ref:`wasting treatment intervention document <intervention_wasting_treatment>`
```

Comment:
Can you add a label at the correct place in the wasting treatment intervention document and then link to that so we know where to look?

Reply:
I am not sure we are set up in this repo to link to subsections on another page, or at least I couldn't figure out how. Instead I added a description of the appropriate section to find the relevant parameter by the name of the table of interest

Reply:
Sphinx lets you add cross-references to arbitrary locations:
https://www.sphinx-doc.org/en/master/usage/restructuredtext/roles.html#cross-referencing-arbitrary-locations

The syntax is the same as the reference you have here that links to the label at the top of the page, but instead you just put a new label at some other location on the page and link to that.

Reply:
Cool, done! Thanks